Common subdirectories: alsa-lib-1.0.27.2.orig/include/alsa and alsa-lib-1.0.27.2/include/alsa
--- alsa-lib-1.0.27.2.orig/include/global.h
+++ alsa-lib-1.0.27.2/include/global.h
@@ -133,18 +133,6 @@
 
 int snd_user_file(const char *file, char **result);
 
-#if !defined(_POSIX_C_SOURCE) && !defined(_POSIX_SOURCE)
-struct timeval {
-	time_t		tv_sec;		/* seconds */
-	long		tv_usec;	/* microseconds */
-};
-
-struct timespec {
-	time_t		tv_sec;		/* seconds */
-	long		tv_nsec;	/* nanoseconds */
-};
-#endif
-
 /** Timestamp */
 typedef struct timeval snd_timestamp_t;
 /** Hi-res timestamp */
--- alsa-lib-1.0.27.2.orig/include/local.h
+++ alsa-lib-1.0.27.2/include/local.h
@@ -30,7 +30,7 @@
 #include <assert.h>
 #include <endian.h>
 #include <stdarg.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <errno.h>
 #include <linux/ioctl.h>
 
Common subdirectories: alsa-lib-1.0.27.2.orig/include/sound and alsa-lib-1.0.27.2/include/sound
--- alsa-lib-1.0.27.2/src/shmarea.c.orig
+++ alsa-lib-1.0.27.2/src/shmarea.c
@@ -19,7 +19,7 @@
  */
   
 #include <stdio.h>
-#include <malloc.h>
+#include <stdlib.h>
 #include <string.h>
 #include <errno.h>
 #include <sys/poll.h>
